example/aws/cdk-lambda/README.template.md

```diff
@@ -0,0 +1,39 @@
+{{ ossHeader }}
+
+# Momento Rust SDK - AWS Lambda CDK Example
+
+This directory contains an example project defining a Rust-based AWS Lambda function that interacts with the Momento Rust SDK.
+The Lambda function is deployed using the AWS Cloud Development Kit (CDK).
+
+## Example Requirements
+
+- Follow the [installation guide](https://doc.rust-lang.org/cargo/getting-started/installation.html) to install Rust and Cargo.
+- You will also need the [cargo-lambda cargo extension](https://www.cargo-lambda.info/)
+- The CDK code in this repo is written in TypeScript, so you will need `Node.js` version 16 or later, and a compatible
+  version of `npm` installed. If you don't have these, we recommend [nodenv](https://github.com/nodenv/nodenv).
+- To get started with Momento you will need a Momento API key. You can get one from the [Momento Console](https://console.gomomento.com).
+
+## Building and Deploying the Lambda Function
+
+To build and deploy the Lambda function, first make sure that your AWS credentials are set up properly (via env vars or
+~/.aws/credentials file). Then all you need to do is run the following commands:
+
+```bash
+cd infrastructure
+npm ci -- deploy --parameters MomentoApiKey=<YOUR_MOMENTO_API_KEY>
```

Comment:
should be `npm ci && npm run cdk` here I think?

Reply:
lol thanks